lib/index.js

```diff
@@ -162,3 +191,15 @@ Ramen.prototype.group = function(group) {
 
   window.Ramen.go();
 };
+
+Ramen.prototype.page = function() {
+  window.Ramen.go();
+};
+
+Ramen.prototype.track = function(track) {
+  window.Ramen.Api.track_named(track.event());
+};
+
+Ramen.version = function() {
```

Comment:
To be a little clearer, I'm thinking something like this (of course, expose it however you like):

```javascript
Ramen.prototype.initialize = function() {
  var self = this;
  this.load(function() {
    window.Ramen.disableURLPolling();
    self.ready();
  });
};
```